src/monitoring_service/cli.py

```diff
@@ -57,9 +65,52 @@ def main(  # pylint: disable=too-many-arguments
     confirmations: BlockNumber,
     min_reward: int,
     debug_shell: bool,
+    accept_disclaimer: bool,
 ) -> int:
     """ The Monitoring service for the Raiden Network. """
     log.info("Starting Raiden Monitoring Service")
+    click.secho(
+        textwrap.dedent(
```

Comment:
Could you put the disclaimer into a variable outside of this function? The code is a bit unpleasant to read with this big text inside it.

Reply:
yes, will do